src/containers/ImageUploader/components/ImageContent.tsx

```diff
@@ -55,6 +56,9 @@ const ImageContent = ({ formik }: Props) => {
               target.files?.[0] ? URL.createObjectURL(target.files[0]) : undefined,
             );
             setFieldValue('imageFile', target.files?.[0]);
+            setFieldValue('contentType', target.files?.[0].type);
+            setFieldValue('fileSize', target.files?.[0].size);
+            setFieldValue('imageDimensions', undefined);
```

Comment:
Dette fungerer, men virker veldig hårete for meg. Innspill, @haattis?
```suggestion
            Promise.resolve(createImageBitmap(target.files?.[0] as Blob).then(image => {
              setFieldValue('imageDimensions', image);
            }));
```